test/mpi/buffer.cc

```diff
@@ -65,6 +72,11 @@ void init_buffer(void *buf, size_t count, ucc_datatype_t dt,
         MPI_Abort(MPI_COMM_WORLD, -1);
         break;
     }
+    if (UCC_MEMORY_TYPE_HOST != mt) {
+        UCC_CHECK(ucc_mc_memcpy(_buf, buf, count * ucc_dt_size(dt),
```

Comment:
maybe use UCC_ALLOC_COPY_BUF as in compare_buffers

Reply:
UCC_ALLOC_COPY_BUF not quite suitable here, `init_buffer()` not required to allocate buf. Please fix me if I'm wrong.